pkg/catalog/catalog.go

```diff
@@ -1492,7 +1493,8 @@ func listDiffHelper(it EntryDiffIterator, prefix, delimiter string, limit int, a
 }
 
 func (c *Catalog) Merge(ctx context.Context, repositoryID string, destinationBranch string, sourceRef string,
-	committer string, message string, metadata Metadata, strategy string) (string, error) {
+	committer string, message string, metadata Metadata, strategy string,
```

Comment:
revert

Reply:
was on auto-merge - I'll revert to oneliner in a different pr.